rtb/verbali/interni/VI_2024-12-23_ver1.0.0.typ

```diff
@@ -0,0 +1,103 @@
+#import "/templates/template.typ": *
+
+#show: content => verbale(
+  data: "23 dicembre 2024",
+  responsabile: "Riccardo Milan",
+  redattori: ("Riccardo Milan",),
+  titolo: "Ottava riunione",
+  verificatori: ("Davide Martinelli",),
+  riassunto: [Riunione interna di fine terzo _sprint_, in cui si è discusso del lavoro svolto e delle attività da svolgere nel quarto _sprint_.],
+  versioni: (
+    "1.0.0",
+    [24/12/2024],
+    "Riccardo Milan",
+    "Davide Martinelli",
+    [- Stesura del verbale],
+  ),
+  content: content,
+)
+
+#infoRiunione(
+  data: "23/12/2024",
+  durata: "2 ore",
+  luogo: "Chiamata Discord",
+  ora: "13:30",
+  partecipantiEsterni: (),
+)
+
+= Sintesi
+Dopo il #rifGlossario("SAL") di fine terzo _#rifGlossario("sprint")_ il gruppo si è riunito per fare il punto della situazione e pianificare il lavoro da svolgere nel quarto _sprint_.
+
+= Ordine del giorno
+All'ordine del giorno i punti fondamentali da svolgere sono relativi sia alla documentazione, ovvero rifinire i casi d'uso e i requisiti e ricercare le metriche di qualità, sia alla realizzazione concreta di un prodotto da presentare come _#rifGlossario("Poc")_.
+
+== Casi d'uso e Requisiti
+Dopo il proficuo incontro svoltosi la settimana precedente con il professor Cardin, in questo _sprint_ si prevede di sistemare tutti i casi d'uso nel documento di Analisi dei Requisiti e, parallelamente, nello stesso documento definire i Requisiti del progetto.
+
+== Metriche
+Continuando il lavoro iniziato nello scorso _sprint_, il gruppo vuole focalizzarsi sulle metriche con cui misurare i progressi del progetto. Per questo sarà importante studiare le metriche di qualità da inserire nelle Norme di Progetto e nel Piano di Qualifica. Riguardo a un primo studio delle metriche svolto nel precedente _sprint_, si è reso necessario codificare uno _script_ che calcola l'#rifGlossario("indice Gulpease").
+
+== PoC
+Lato pratico il punto della situazione è che bisogna ultimare la transizione da #rifGlossario("Apache Nifi") alla costruire un servizio _custom_ di _data #rifGlossario("stream processing")_ utilzzando il _framework_ #rifGlossario("Spring"), popolare il _database_ con dei dati di _test_ e aggiornare l'interfaccia _#rifGlossario("Grafana")_ per ricevere correttamente i dati dal nuovo _stram processor_.
```

Comment:
...da Apache NiFi a un servizio custom di ... costruito utilizzando il framework Spring

Errore di battitura nell'ultima parola, c'è scritto stream senza la e

Se vuoi puoi mettere queste tre cose in un elenco puntato ma va bene anche così visto che c'è la tabella decisioni alla fine